src/main/java/TasksLoader.java

```diff
@@ -0,0 +1,100 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TasksLoader {
+    private TasksList tasksList;
+
+    public TasksLoader(TasksList tasksList) {
+        this.tasksList = tasksList;
+    }
+
+    public void loadTasks() {
```

Comment:
This method is quite long. You might want to separate into smaller ones, it just a good practice to have